app/controllers/admin/lead_provider_users_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class Admin::LeadProviderUsersController < Admin::BaseController
+  skip_after_action :verify_policy_scoped
+  before_action :load_lead_provider, only: %i[edit update]
+  before_action :load_lead_provider_user, only: %i[edit update]
+
+  def edit
+    authorize User
+  end
+
+  def update
+    authorize User
+
+    if @lead_provider_user.update(user_params)
+      redirect_to :admin_supplier_users, notice: "Changes saved successfully"
+    else
+      render :edit
+    end
+  end
+
+private
+
+  def load_lead_provider
+    @lead_provider = LeadProvider.find(params[:lead_provider])
+  end
+
+  def load_lead_provider_user
+    @lead_provider_user = User.find(params[:id])
+  end
+
+  def user_params
+    params.require(:user).permit(:full_name, :email)
+  end
```

Comment:
@ethanmills that's the repetition I was thinking about - if admins can edit all users, we will have a ton of places with these types of parameters :/

Reply:
Perhaps Pundit's `permitted_attributes`? https://github.com/varvet/pundit#strong-parameters

Reply:
Specifically: https://github.com/varvet/pundit#user-content-strong-parameters:~:text=If%20you%20want%20to%20permit%20different,a%20permitted_attributes_for_%23%7Baction%7D%20method%20on%20your%20policy

Reply:
holy crap, that sounds exactly like what would solve it